.github/workflows/build.yaml

```diff
@@ -31,20 +31,10 @@ jobs:
           path: zipline/src/jvmMain/resources/jni
           if-no-files-found: error
 
-  platform-tests:
+  jvm-test-binary:
```

Comment:
This shard builds the JVM tests once, and they embed the necessary JS files in resources.